app/services/task_scheduler.py

```
@@ -794,9 +794,19 @@ def recover_schedules(self) -> int:
+    On first call, recovers persisted schedules from the database.
+    """
     global _scheduler_instance
     if _scheduler_instance is None:
         supabase = get_supabase_client()
         _scheduler_instance = TaskScheduler(supabase)
+        # Recover persisted schedules on startup
+        try:
+            recovered = _scheduler_instance.recover_schedules()
+            if recovered > 0:
+                logger.info(f"Recovered {recovered} scheduled tasks from database")
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)